docs/global_constant.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Global constants
+author: Roxane Letourneau
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Description of the `Global Constants` feature
+
+The `Global Constants` is a feature introduced in the Hangzhou protocol. Its goal is to allow users to originate larger contracts or share code between contracts by using registered constants. 
```

Comment:
Here you could write instead

The `Global Constants` feature is introduced in the Hangzhou protocol.

Reply:
I'll make the change, thanks for the suggestion.